src/components/Map/index.tsx

```diff
@@ -58,16 +130,58 @@ const Map = ({
                       }
                     : undefined
             }
+            ref={mapRef}
         >
-            {scooters?.map((scooter) => (
-                <Marker
-                    key={scooter.id}
-                    latitude={scooter.lat}
-                    longitude={scooter.lon}
-                >
-                    <ScooterOperatorLogo logo={scooter.operator} size={24} />
-                </Marker>
-            ))}
+            {clusters.map((scooterCluster) => {
+                // every cluster point has coordinates
+                const [
+                    slongitude,
+                    slatitude,
+                ] = scooterCluster.geometry.coordinates
+                // the point may be either a cluster or a scooterpoint
+                const { cluster: isCluster } = scooterCluster.properties
+                let pointCount = 0
+                // the point can have clusterProperties or scooterProoperties
+                if (scooterCluster.properties.cluster)
+                    pointCount = (scooterCluster.properties as ClusterProperties)
+                        .point_count
+                // we have a cluster to render
+                if (isCluster) {
```

Comment:
Kanskje lage ein `ScooterMarker`-komponent som tar `pointCount` som prop og rendre cluster/marker basert på det?

Reply:
Endret slik at valget mellom cluster og ScooterOperatorLogo skjer i den komponenten også

Propen `scooterCluster.properties.point_count` eksisterer ikke om punktet ikke er et cluster forresten. Den if'en hvor castinga til `ClusterProperties` skjer burde kanskje bare inneholde `isCluster` for å gjøre det litt mer leselig.